### PR TITLE
update 18-9-2023

### PR DESCRIPTION
**Pull request by Dobby-progam**
- some artilce writer update
- created about us
- some clean up

### CODE REVIEW

File: src/styling/articleWriter.css [hunk 1]
```diff
@@ -1,33 +1,41 @@
...
-
 textarea#summary {
-    width: 400px;
+    width: 200%;
     height: 100px;
 }
 
-textarea#title {
-    width: 400px;
+textarea#content {
```
Comment (JoshuaSlui) [inline src/styling/articleWriter.css:12]: I think for consistency, it's best if you apply this width to all the fields.

**Review by JoshuaSlui — CHANGES_REQUESTED** [review]

**Comment by JoshuaSlui** [comment]
And while you are at it, can you focus on bug-point#2 as well?
> Approve/deny buttons dont inline on /reviewing

And perhaps additions-point#7?
> Add blockquote styling

File: src/styling/articleWriter.css (hunk 1 above)
Comment (Dobby-progam) [reply src/styling/articleWriter.css:12]: https://github.com/Wolfsvale/Aternos-guides/pull/8/commits/98989d2d467ddf1c94f9f78a72c00bf1889d6d02

Comment (Dobby-progam) [reply src/styling/articleWriter.css:12]: should be fixed

**Comment by Dobby-progam** [comment]
> And while you are at it, can you focus on bug-point#2 as well?
> 
> > Approve/deny buttons dont inline on /reviewing

fixed in https://github.com/Wolfsvale/Aternos-guides/pull/8/commits/25b2345cb468c1ec0fd3a8866d78c776bf7a119b

**Review by JoshuaSlui — APPROVED** [review]

**Pull request merged**